pkg/trace/writer/sender.go

```diff
@@ -351,6 +351,16 @@ func newPayload(headers map[string]string) *payload {
 	return p
 }
 
+func (p *payload) clone() *payload {
+	headersClone := make(map[string]string, len(p.headers))
```

Comment:
This name is a bit too long. Can we please just use `headers`. We're generally following the principle of:

> The greater the distance between a name's declaration and its uses, the longer the name should be.
- [What's in a name](https://talks.golang.org/2014/names.slide#4) by Andrew Gerrand

In this case the distance is very short, so `headers` or `h` is best.